README.md

```diff
@@ -11,21 +11,24 @@
 
 ## Overview
 
-The **Azure IoT Device Workbench** is a Visual Studio Code extension that provides an integrated environment to code, build, deploy, and debug your IoT device project with multiple Azure services supported. The extension also supports working with [IoT Plug and Play](https://docs.microsoft.com/azure/iot-pnp/overview-iot-plug-and-play) by defining *device capability model* schemas and generating skeleton device code and projects.
+The **Azure IoT Device Workbench** is a Visual Studio Code extension that provides an integrated environment to code, build, deploy, and debug your IoT device project with multiple Azure services supported. 
 
 ## Get Started
 
 ### Develop device using IoT Plug and Play
 
-[IoT Plug and Play <sup>public preview</sup>](https://docs.microsoft.com/azure/iot-pnp/) is a *device capability model schema* that describes device capabilities. The Azure IoT Device Workbench extension provides an integrated environment to author IoT Plug and Play *device capability models (DCM)* and *interfaces*, publish to model repositories, and generate skeleton C code to implement the device application.
+[**IoT Plug and Play <sup>preview refresh</sup>**](https://docs.microsoft.com/azure/iot-pnp/) is a *device capability model schema* that describes device capabilities. Learn how to get started with IoT Plug and Play:
```

Comment:
Since IoT PnP has been deprecated from workbench. Maybe we can put this introduction section behind the "Develop in generic device" section?

Reply:
Add brief introduction of PnP Preview Refresh in Overview section, and move the PnP section to the last of Get Started